### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by snowly1** [Y4hL/PyDoor#12] [General]
Could not find where to change port/ip both client.py and server-cli.py would you share the variables to change? 

Regards

**Comment by tidely** [comment]
Ah I should probably be more clear about where it is. Thanks

At the bottom of both files, a tuple is passed into the main loop. It should look like this: ('', 8001) '' is the ip address and 8001 is the port used.

**Comment by snowly1** [comment]
Thanks, found it. Will try it out!
